README.md

```diff
@@ -37,6 +37,12 @@ TODO: Using the "tree -d" command can be a helpful way to generate this informat
 **{list directories and descriptions}**
 -->
 
+## Development and Software Delivery Lifecycle
+We follow GitHub Flow with protected branches and pull request reviews. Development includes automated code analysis, security scanning, and adherence to CMS Open Source Policy guidelines. See [CONTRIBUTING.md](CONTRIBUTING.md) for more details.
+
+## Local Development
+To run locally, please follow the instructions in [CONTRIBUTING.md](CONTRIBUTING.md) under Buidling the Project and Building Dependencies.
+
 
 ## Usage
 
```

Comment:
It looks like this used an old README.md template so the "Policies" section below is outdated. Please update to: https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/README.md#policies

Reply:
plz fix upstream as well 🔧

Reply:
fixed upstream already!